packages/lib-react-components/src/ZooFooter/ZooFooter.js

```diff
@@ -49,7 +54,20 @@ const defaultProps = {
     'https://status.zooniverse.org/',
     'https://www.zooniverse.org/security'
   ],
-  projectNavListURLs: ['https://www.zooniverse.org/projects'],
+  projectNavListURLs: [
```

Comment:
This urls are duplicating `https://www.zooniverse.org/projects` ?

i.e. I think `'https://www.zooniverse.org/projectshttps://www.zooniverse.org/projects?discipline=arts'` should just be `'https://www.zooniverse.org/projects?discipline=arts'`.